powerhome/playbook
# [NUXE-305] - Link kit and sample pages

#### What does this PR do?

1. Limits all kits page to single doc example.  Creates secret route at `/all_kit_examples` which still renders out every kit with every doc example.

2. Makes bi-directional connection between kit and sample pages.  Kit's have links to all sample pages that make use of them, and sample pages have links to all the kits they use.

#### Screens

<img width="632" alt="cap4" src="https://user-images.githubusercontent.com/57693937/101046314-f0e20e80-354e-11eb-8cf9-71e0c05ef849.png">
<img width="452" alt="cap1" src="https://user-images.githubusercontent.com/57693937/101046013-b5dfdb00-354e-11eb-8968-dc68e032e03f.png">
<img width="482" alt="cap2" src="https://user-images.githubusercontent.com/57693937/101046014-b5dfdb00-354e-11eb-8736-39efea07928d.png">

#### Breaking Changes

No, but check out Nitro's all playbook kits page in dev_docs just in case.  Changes were explicitly made to be backwards compatible but haven't checked yet in Nitro.

#### Runway Ticket URL

[Link to Runway Story 1](https://nitro.powerhrg.com/runway/backlog_items/NUXE-305)
[Link to Runway Story 2](https://nitro.powerhrg.com/runway/backlog_items/NUXE-62)

#### How to test this

Check out the sample and kits pages and see if they are displaying links to each other appropriately.  Check `/kits` to see if it's limited to one example each and finally check the new secret route `/all_kit_examples` to see if that's displaying all kits (the original functionality).

#### Checklist:


*The normal release cut off deadline is 3p EDT each week. Please reach out to the release team if you have an urgent request or need a release off cycle.*
